docs/UserGuide.md

```diff
@@ -1,134 +1,340 @@
-6. Refer to the [Features](#features) section below for details of each command.<br>
+7. Refer to the [Features](#features) section below for details of each command.<br>
+
 
 
-## Features
+## **Features**
 
-> **Command Format**
-> * Words in `UPPER_CASE` are the parameters.
-> * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+All the features in Inbx_0 are performed by typing in commands in the command line. There is a certain command format for each command and the general format can be described as such: 
 
-#### Viewing help : `help`
+**Command Format**
+1.  Words in `UPPER_CASE` are the parameters that are compulsory and will be taken in by the app (e.g. NAME, DATE). 
+<br>
+2. Items in `SQUARE_BRACKETS` are optional and will not be required in order to perform the command (e.g. [tags], [date]).
+<br>
+3. Items with `...` after them can have multiple instances. This means that there can be more than one occurrence of the particular item. For example, “[t/TAG]...” means that you can have multiple tags. 
+<br>
+4.  The order of parameters is fixed and needs to be strictly followed.
+
+The rest of the guide will be using the general command format to describe what is needed to execute each command.
+
+#### <a id="help"></a>Help Command
 Format: `help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+Looking for help? If you ever need a reminder on how to input certain commands or have some troubleshooting issues, you can easily access the ‘help’ command which will guide you to the right direction.
+
+To access the help command, type the following into the command line:
+> help
+
+This will show a list of available commands with detailed instructions on how to execute each command and any issues that you might face.
+
+> The Help command will be also activated if you enter an incorrect or invalid command repeatedly e.g. `abcd`
+
+<br><br>
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### <a id="add"></a>Adding a task: `add`
+Adds a floating task.<br>
+Format: `add TASK` 
 
-> Persons can have any number of tags (including 0)
+> Floating tasks are tasks without deadlines.
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `add Buy Groceries` <br>
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
+Adds a task with deadlines.<br>
+Format: `add TASK e/END_DATE, END_TIME i/IMPORTANCE, [t/TAGS]` 
+
+Examples: 
+* `add Do CS2103 Homework e/tomorrow, 10am, i/red`
+* `add Finish Project Paper e/1 March, 12am i/green` <br>
+
+Adds an event.<br>
+Format: `add TASK s/START_DATE, START_TIME e/END_DATE, END_TIME i/IMPORTANCE`
+
+Parameters: <br>
```

Comment:
Based on your description above, all words in upper_case are compulsory. Isn't it too restrictive to make importance and tags compulsory? Plus,  start date/time and  end date/time might not be needed based on the type of task you add.